frontend/src/components/ProjectList.tsx

```diff
@@ -0,0 +1,44 @@
+import { useState, useEffect } from "react";
+import useAuth from "../hooks/Auth.tsx";
+import "../styles/ProjectList.css";
+
+interface Project {
+    projectid: string;
+    name: string;
+    description: string;
+}
```

Comment:
This should probably moved to another file to be used by other components (ViewProject for example)